contracts/mixnet/src/contract.rs

```diff
@@ -42,6 +42,8 @@ pub const DEFAULT_SYBIL_RESISTANCE_PERCENT: u8 = 30;
 // We'll be assuming a few more things, profit margin and cost function. Since we don't have reliable package measurement, we'll be using uptime. We'll also set the value of 1 Nym to 1 $, to be able to translate epoch costs to Nyms. We'll also assume a cost of 40$ per epoch(month), converting that to Nym at our 1$ rate translates to 40_000_000 uNyms
 pub const DEFAULT_COST_PER_EPOCH: u32 = 40_000_000;
 
+pub const VESTING_CONTRACT_ADDR: &str = "";
```

Comment:
Yeah, I don't have a good take on here, we should probably do what ever feels more ergonomic, we can also put it in contract state, so that we can update it more easily

Reply:
I don't think we even need a vesting address here, since we control on the proxy, noone can undelegate/unbond except the proxy that did the original bonding/delegation, so there is no specific need to further gate on the vesting contract, as far as we care anyone can delegate on behalf of someone else, as long as they pay the bonding/delegation fee...